### PR TITLE
clean up output of EasyBuild bootstrap script & add version

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/scripts/bootstrap_eb.py [hunk 1]
```diff
@@ -110,6 +114,32 @@ def error(msg, exit=True):
...
+
+
+def restore_stdout_stderr(orig_stdout, orig_stderr):
+    """Restore stdout/stderr channels after mocking"""
+    sys.stdout.flush()
+    stdout = sys.stdout.getvalue()
+    sys.stderr.flush()
+    stderr = sys.stderr.getvalue()
+
+    sys.stdout.flush()
```
Comment (wpoely86) [inline easybuild/scripts/bootstrap_eb.py]: why flush twice?

Comment (boegel) [inline easybuild/scripts/bootstrap_eb.py]: good point, I'll clean this up

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**